setup.py

```diff
@@ -50,7 +50,7 @@ def get_install_requires():
     download_url='https://github.com/TheOneAboveAllTitan/openwisp-notifications/releases',
     platforms=['Platform Independent'],
     keywords=['django', 'netjson', 'notification', 'openwisp', 'monitoring'],
-    packages=find_packages(exclude=['tests', 'docs']),
+    packages=find_packages(exclude=['tests*', 'docs*']),
```

Comment:
I'm aware that @nemesisdesign made this recommendation but I don't see any difference

Reply:
Any idea how to not include

Reply:
I don't think the goal was to remove these, but to remove test project. 

> I found out that most modules include the test project in the python package, which is the installed in .../lib/site-packages/tests (very wrong)

-@nemesisdesign

Reply:
Yeah, but I think we can go one step forward and remove them too. If you are unsure, let's wait for @nemesisdesign's opinion

Reply:
tests should be included because they can be ran, we run some tests in ansible-openwisp2 as part of the automated build, for example. It's not recommended to remove them although now I can't quickly find the source where I read that.